tasks/install_hashi_repo.yml

```diff
@@ -0,0 +1,32 @@
+---
+# File: tasks/install_hashi_repo.yml
+#       Install Vault via HashiCorp Linux repository
+
+- name: Add HashiCorp yum repo
+  shell: yum-config-manager --add-repo=https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo
```

Comment:
**ansible-lint**: 305: Use `shell:` only when shell functionality is required. Using `command:` is prefered.

Reply:
Fixed. Thanks for pointing it out!